README.md

```diff
@@ -61,11 +61,15 @@ mas "1Password", id: 443987910
 whalebrew "whalebrew/wget"
 ```
 
-## Note
+## Lockfile
 
-Homebrew does not support installing specific versions of a library, only the most recent one, so there is no good mechanism for storing installed versions in a `.lock` file.
+Homebrew does not support installing specific versions of a library, only the most recent one, so there is no mechanism for installing specific versions given only a `.lock` file.
 
-If your software needs specific versions then perhaps you'll want to look at using [Vagrant](https://vagrantup.com/) to better match your development and production environments.
+If your software needs specific versions, consider [Vagrant](https://vagrantup.com/), Docker (perhaps with [whalebrew](https://github.com/whalebrew/whalebrew), which homebrew-bundle supports), or another tool to install versions explicitly.
+
+After bundle installation, homebrew-bundle creates a `Brewfile.lock.json`. This file is for information purposes only as a record of the last successful `brew bundle`. Its presence does not change the result or behavior of `brew bundle`, but it can be useful as a record of a previous success when a subsequent `brew bundle` fails. It's probably not useful to commit to `git` or other version control, as it can contain local system info that varies between systems.
```

Comment:
```suggestion
After a successful `brew bundle` run, it creates a `Brewfile.lock.json` to record the environment. If a future `brew bundle` run fails, you can check the differences between `Brewfile.lock.json` to debug. As it can contain local environment information that varies between systems, it not worth committing to version control on multi-user repositories.
```